audonnx/core/function.py

```diff
@@ -0,0 +1,59 @@
+import typing
+
+import numpy as np
+
+import audobject
+
+
+class Function(audobject.Object):
+    r"""Turn function into an :class:`audobject.Object`.
```

Comment:
Does this make sense as part of `audonnx`, or would it be better located in `audobject`?

Reply:
The only downside is that `librosa` is not automatically installed with `auto_install=True`. I was already thinking if it makes sense to add support for additional dependencies in `audobject`. But that is something we would need to implement in `audobject`.